src/main/java/com/codeborne/selenide/Condition.java

```diff
@@ -272,10 +273,15 @@ public static Condition matchText(String regex) {
    * <p>NB! Ignores multiple whitespaces between words</p>
    *
    * @param text expected text of HTML element
+   *
+   * @throws IllegalArgumentException if given text is null or empty
    */
   @CheckReturnValue
   @Nonnull
   public static Condition text(String text) {
+    if (StringUtils.isBlank(text)) {
+      throw new IllegalArgumentException("No expected text given");
```

Comment:
I will be nice to have more meaningful error message. E.g. "Text condition must not be empty string." or smth else which explain the problem with compating actual text and the empty string.

Reply:
Ok. I changed it